src/changelog/2024/08/static-file-service-navigation-visibility.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Static File Service Navigation and Visibility
+description: Navigate static files with better navigation and visibility selector
+date: 2024-09-19 13:00:00.0
+authors: ["serban-costin"]
+tags:
+    - changelog
+---
+
+Users of FlowFuse Cloud and self-hosted enterprise users can now make use of an enhanced navigation tool and visibility selector when using the static file service feature.
+
+Starting with the v2.9.0 release self-hosted enterprise and FlowFuse Cloud Team tier users have the ability to set the visibility of their static files through a new navigation tool.
+
+This update means that users can set the visibility of their uploaded files to public and make them accessible outside the node-red instance itself.
+
+This capability enhances the current static file service which only made uploaded files accessible to the node-red instance 
+itself while also allowing users to set a static path map on which the node-red instance will expose the files. 
+
+Considerations:
+- Visibility and static path maps can be set on folders only.
+- Any change in visibility settings require an instance restart in order for the changes to take effect.
+- URL's and file paths are copiable
```

Comment:
```suggestion
Users of FlowFuse Cloud can now make use of an enhanced navigation tool and visibility selector when using the [static file service feature](https://flowfuse.com/blog/2024/08/flowfuse-2-8-release/#static-assets-service).

By settings the visibility of a folder to 'public', their Node-RED instance will automatically make any files in that folder publicly available - on a path of the user's choosing.

This makes it very simple to serve up additional assets such as images or libraries that can be used in your Node-RED Dashboards.

The visibility settings apply to the folder; you cannot make individual files public. Any changes to the visibility settings will require the instance to be restarted for the changes to take effect.
```

Reply:
Slight rewording to add a bit more on what this enables for the user.

Reply:
thank you!